test/Module/Command/DeregisterTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-tooling for the canonical source repository
+ * @copyright Copyright (c) 2017 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-tooling/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive\Tooling\Module\Command;
+
+use Mockery;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
```

Comment:
Hm... this is recommended in Mockery documentation:
http://docs.mockery.io/en/latest/reference/phpunit_integration.html

I was trying to remove `@runInSeparateProcess` but some tests on some versions (php versions/dependencies) were failing. As you can note in some tests we still have this annotation.